src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -86,6 +88,11 @@ export const CurationPage = (props: ICurationPageProps) => {
     }
     if (geneEntity && props.firebaseInitSuccess) {
       const cleanupCallbacks: Unsubscribe[] = [];
+      cleanupCallbacks.push(
+        onValue(ref(props.firebaseDb, firebaseMetaCurrentReviewerPath), snapshot => {
+          props.setReadOnly(snapshot.val() ? true : false);
```

Comment:
Can keep it concise using double negation
```suggestion
          props.setReadOnly(!!snapshot.val());
```